src/embind/embind.js

```diff
@@ -1191,10 +1187,7 @@ var LibraryEmbind = {
         var setter = field.setter;
         var setterContext = field.setterContext;
         fields[fieldName] = {
-          read: (ptr) => {
-            return getterReturnType['fromWireType'](
-                getter(getterContext, ptr));
-          },
+          read: (ptr) => getterReturnType['fromWireType'](getter(getterContext, ptr)),
           write: (ptr, o) => {
```

Comment:
I think these two might make more sense using method syntax?

Reply:
Idk, maybe just splitting into lines is enough?